Mathlib/Geometry/Manifold/VectorBundle/GramSchmidtOrtho.lean

```diff
@@ -0,0 +1,270 @@
+/-
```

Comment:
I'm not sure about the location of the file: there is nothing smooth in this file, so shouldn't it go in `Topology/VectorBundle`?

Reply:
That's fair: moved to `Topology/VectorBundle`.